gamemodes/terrortown/entities/weapons/weapon_ttt2_lens.lua

```diff
@@ -335,38 +292,38 @@ else
 		end
 	end
 
-	-- performance improvements
-	function SWEP:Deploy()
+	function SWEP:ShowFootPrints(show)
 		local owner = self:GetOwner()
-
-		if not hook_installed and IsValid(owner) and owner == LocalPlayer() and owner:Alive() and owner:IsTerror() then
-			hook.Add("PostDrawTranslucentRenderables", "TTT2SnifDrawFootSteps", DrawFootsteps)
-			hook.Add("Think", "TTT2UpdateFootsteps", UpdateFootsteps)
-
-			hook_installed = true
+		if not IsValid(owner) then
+			return true
 		end
 
-		return BaseClass.Deploy(self)
-	end
+		if show then
+			if not hook_installed and IsValid(owner) and owner == LocalPlayer() and owner:Alive() and owner:IsTerror() then
+				hook.Add("PostDrawTranslucentRenderables", "TTT2SnifDrawFootSteps", self.DrawFootsteps)
+				hook.Add("Think", "TTT2UpdateFootsteps", self.UpdateFootsteps)
 
-	function SWEP:Holster()
-		local owner = self:GetOwner()
+				hook_installed = true
+			end
+		else
+			if hook_installed and owner == LocalPlayer() then
+				hook.Remove("PostDrawTranslucentRenderables", "TTT2SnifDrawFootSteps")
+				hook.Remove("Think", "TTT2UpdateFootsteps")
 
-		if not IsValid(owner) then
-			return true
+				hook_installed = false
+			end
 		end
+	end
 
-		if hook_installed and owner == LocalPlayer() then
-			hook.Remove("PostDrawTranslucentRenderables", "TTT2SnifDrawFootSteps")
-			hook.Remove("Think", "TTT2UpdateFootsteps")
+	-- performance improvements
+	function SWEP:Deploy()
+		-- self:ShowFootPrints(true)
 
-			hook_installed = false
-		end
+		return BaseClass.Deploy(self)
+	end
 
-		local vm = owner:GetViewModel()
-		if IsValid(vm) then
-			self:ResetBonePositions(vm)
-		end
+	function SWEP:Holster()
+		-- self:ShowFootPrints(false)
```

Comment:
is there a reason this is commented out?

Reply:
no, just had it marked in case i didn't like the behavior of only seeing footprints when zoomed